searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp

```diff
@@ -575,6 +580,29 @@ StoreOnlyFeedView::internalRemove(FeedToken token, const RemoveOperation &rmOp)
     }
 }
 
+void
+StoreOnlyFeedView::internalRemove(FeedToken token, const RemoveOperationWithGid &rmOp)
+{
+    assert(rmOp.getValidNewOrPrevDbdId());
+    assert(rmOp.notMovingLidInSameSubDb());
+    const SerialNum serialNum = rmOp.getSerialNum();
+    DocumentId dummy;
+    PendingNotifyRemoveDone pendingNotifyRemoveDone = adjustMetaStore(rmOp, rmOp.getGlobalId(), dummy);
+    considerEarlyAck(token);
+
+    if (rmOp.getValidDbdId(_params._subDbId)) {
```

Comment:
The expression for this if should never evaluate to true.

Reply:
Removed Section.